src/utils/shortLink.ts

```diff
@@ -0,0 +1,89 @@
+import { ShortLinkPlatform as ShortLinkPlatformType } from '@graasp/sdk';
+
+import { BUILDER } from '@/langs/constants';
+
+// TODO: check where to store them
+export const MIN_SHORT_LINK_LENGTH = 6;
+export const MAX_SHORT_LINK_LENGTH = 255;
+
+export function randomString(length: number): string {
+  const characters =
+    'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+  let rand = '';
+
+  for (let i = 0; i < length; i += 1) {
+    const randomIndex = Math.floor(Math.random() * characters.length);
+    rand += characters.charAt(randomIndex);
+  }
```

Comment:
```suggestion
  Array.from({length}, ()=> {
  const randomIndex = Math.floor(Math.random() * characters.length);
    return characters.charAt(randomIndex)
  }).join('')
```
or something similar?